<launch>
    <node pkg="camera_node" type="camera_node" name="camera_node" output="log" ns="camera_node/front">
        <param name="frame_rate" value="10"></param>
        <param name="camera_port" value="0"></param>
        <!-- <param name="video_path" value="$(find camera_node)/../../data/src_videos/reverse_short.mp4"> </param> -->
        <!-- <param name="video_path" value="$(find camera_node)/../../data/src_videos/reverse_stages2.mp4"> </param> -->
        <param name="video_path" value="$(find camera_node)/../../data/src_videos/bottom.avi"> </param>
        <!-- <param name="video_path" value="$(find camera_node)/../../data/src_videos/reverse_stages.avi"> </param> -->
        <remap from="/image_topic" to="/image_topic/front"/>
    </node>
    <node pkg="path" type="forward_park" name="fordward_park" output="screen"> </node>
    <param name="exit_park" value="1"> </param>
    <rosparam file="$(find path)/launch/times.yaml"></rosparam>
    <!-- float -->
</launch>
